docs/docs-content/clusters/edge/site-deployment/kubevip.md

```diff
@@ -0,0 +1,111 @@
+   ```yaml
+   cluster:
+    kubevipArgs:
+      vip_interface: "INTERFACE_NAME"
+      svc_enable: true
+      vip_servicesinterface: "INTERFACE_NAME"
+   ```
+
+   These are kube-vip environment variables that enable kube-vip to provide load balancing services for Kubernetes services and specify which network interface will be used by kube-vip for handling traffic to the Kubernetes API server and Kubernetes services. The following table provides you guidance on how to choose the values for each parameter:   
+ 
+  | **Parameter** | **Description** |
+  |-----------|-------------|
+  | `vip_interface` | Specifies the Network Interface Controller (NIC) that kube-vip will use for handling traffic to the Kubernetes API. If you do not specify `vip_serviceinterface`, kube-vip will also use this interface for handling traffic to LoadBalancer-type services. |
+  | `svc_enable` | Enables kube-vip to handle traffic for services of type LoadBalancer. |
+  | `vip_serviceinterface` | Specifies the NIC that kube-vip will use for handling traffic to LoadBalancer-type services. If your cluster has network overlay enabled, or if your host has multiple NICs and you want to publish services on a different NIC than the one used by Kubernetes, you should specify the name of the NIC as the value of this parameter. If this parameter is not specified and you have set `svc_enable` to `true`, kube-vip will use the NIC you specified in `vip_interface` to handle traffic to LoadBalancer-type services.  |
+
+8. Next, in layer of your cluster profile that has the service you want to expose, add two parameters `loadBalancerIP: 0.0.0.0` and `loadBalancerClass: kube-vip.io/kube-vip-class` to the service spec. The following example manifest displays the usage of these two parameters.
+   ```
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: http-app-svc
+      namespace: myapp
+    spec:
+      loadBalancerIP: 0.0.0.0
+      loadBalancerClass: kube-vip.io/kube-vip-class
+      ports:
+      - port: 80
+        protocol: TCP
+        targetPort: http
+      selector:
+        app.kubernetes.io/name: http-app
+      type: LoadBalancer
+   ```
+
+9. Click **Next** and finish the rest of the configurations. For more information, refer to [Create Cluster Definition](./site-installation/cluster-deployment.md).
+
+When the cluster finishes deploying, kube-vip adds an annotation named `kube-vip.io/requestedIP` to the Service resource to document which IP address it has received from the external network. Whenever kube-vip restarts, it will attempt to re-request the same IP address for that service. You can remove the annotation to make kube-vip request a fresh address with the following command. Replace `SERVICE_NAME` with the name of your service, and make sure to include the minus symbol`-` symbol at the end of the annotation. 
```

Comment:
```suggestion
When the cluster finishes deploying, kube-vip adds an annotation named `kube-vip.io/requestedIP` to the Service resource to document which IP address it has received from the external network. Whenever kube-vip restarts, it will attempt to re-request the same IP address for that service. You can remove the annotation to make kube-vip request a fresh address with the following command. Replace `SERVICE_NAME` with the name of your service, and make sure to include the minus symbol`-` at the end of the annotation. 
```